src/main/java/Duke.java

```diff
@@ -1,10 +1,117 @@
+import java.util.*;
+
 public class Duke {
+
+    public static final String SEPARATING_LINE = "  ________________________________________";
+    public static final int MAX_TASKS = 100;
+    public static final int TODO_LEN = 5;
+    public static final int EVENT_LEN = 6;
+    public static final int DEADLINE_LEN = 9;
+    public static int currentTaskCount = 0;
+    static Task[] tasksToDo = new Task[MAX_TASKS];
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        greet();
+        GetTask();
+        bye();
+    }
+
+    public static void greet(){
```

Comment:
You might want to shift some of the message codes into another class called messages so that there would not be too many function in a single class. I think this would help greatly with code readability